playbooks/robusta_playbooks/alerts_integration.py

```diff
@@ -261,20 +264,48 @@ def logs_enricher(event: PodEvent, params: LogEnricherParams):
             )
         return
 
+    containers: list[str] = []
+    all_statuses = pod.status.containerStatuses + pod.status.initContainerStatuses
+    if params.container_name:
+        containers = [params.container_name]
+    elif event.get_subject().container:
+        # support alerts with a container label, make sure its related to this pod.
+        containers = [
+            container_status.name
+            for container_status in all_statuses
+            if container_status.name == event.get_subject().container
+        ]
+
+    # If no container is specified, find containers in an unhealthy state. good for pending pods
+    if len(containers) == 0:
+        containers = [
+            container_status.name
+            for container_status in all_statuses
+            if container_status.state.waiting is not None and container_status.restartCount >= 1
+        ]
+
+    tries: int = 2
+    backoff_seconds = 2
     regex_replacement_style = (
         RegexReplacementStyle[params.regex_replacement_style] if params.regex_replacement_style else None
     )
-    log_data = pod.get_logs(
-        regex_replacer_patterns=params.regex_replacer_patterns,
-        regex_replacement_style=regex_replacement_style,
-        previous=params.previous,
-    )
-    if not log_data:
-        return
+    for container in containers:
```

Comment:
I think this breaks the UI logs
The UI logs uses the `logs_enricher` action
The UI doesn't send container atm, the PodEvent is created without a container, and often nothing is crashing.
On such a scenario, `containers` will be empty, and we'll have no logs.

I think the default behavior, should be like before, which is to get the logs of the first container, or get logs of all containers

If we return more than one file block, we need to verify the UI keeps working as before